test/test_mcs.py

```diff
@@ -0,0 +1,136 @@
+"""
+Regression tests for the MCS class
+
+These might start failing if RDKit changes canonical order of atoms.
+"""
+import pkg_resources
+import pytest
+from rdkit import Chem
+
+from lomap import mcs
+
+
+def _rf(fn):
+    # get path to file from inside lomap installation
+    f = pkg_resources.resource_filename('lomap', 'test/' + fn)
+    return f.replace('/lomap/test', '/test')
+
+
+@pytest.fixture
+def toluene():
+    f = _rf('basic/toluene.mol2')
+
+    return Chem.MolFromMol2File(f)
+
+
+@pytest.fixture
+def methylcyclohexane():
+    f = _rf('basic/methylcyclohexane.mol2')
+
+    return Chem.MolFromMol2File(f)
+
+
+@pytest.fixture
+def methylnaphthalene():
+    f = _rf('basic/2-methylnaphthalene.mol2')
+
+    return Chem.MolFromMol2File(f)
+
+
+@pytest.fixture
+def trimethylnaphthalene():
+    f = _rf('basic/1,3,7-trimethylnaphthalene.mol2')
+
+    return Chem.MolFromMol2File(f)
+
+
+@pytest.fixture
+def butylmethylbenzene():
+    f = _rf('basic/1-butyl-4-methylbenzene.mol2')
+
+    return Chem.MolFromMol2File(f)
+
+
+@pytest.fixture
+def dimethylnaphthalene():
+    f = _rf('basic/2,6-dimethylnaphthalene.mol2')
+
+    return Chem.MolFromMol2File(f)
+
+
+@pytest.fixture
+def methylpropylnaphthalene():
+    f = _rf('basic/2-methyl-6-propylnaphthalene.mol2')
+
+    return Chem.MolFromMol2File(f)
+
+
+@pytest.fixture
+def naphthol():
+    f = _rf('basic/2-naftanol.mol2')
+
+    return Chem.MolFromMol2File(f)
+
+
+def test_toluene_to_methylcyclohexane(toluene, methylcyclohexane):
+    mapper = mcs.MCS(toluene, methylcyclohexane)
+
+    mapping = mapper.heavy_atom_mcs_map()
+
+    assert sorted(mapping) == [(0, 0), (1, 1), (2, 6), (3, 5), (4, 4), (5, 3),
+                               (6, 2)]
+
+
+def test_toluene_to_methylnaphthalene(toluene, methylnaphthalene):
+    mapper = mcs.MCS(toluene, methylnaphthalene)
+
+    mapping = mapper.heavy_atom_mcs_map()
+
+    assert len(mapping) == 7
```

Comment:
Is there something specific about this chemistry where we want to check the length of the mapping, but don't the other maps?